ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -276,3 +276,7 @@ def greater_than(self, y, name=None):
     @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
     def rsqrt(self, name=None):
         return ivy.reciprocal(ivy.sqrt(self._ivy_array))
+
+    @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")

```

Comment:
Thanks for contributing! LGTM, minor thing: Could this be modified to 2.5.0

Reply:
Done